submissions/kt-std/js-dom/js/script.js

```diff
@@ -0,0 +1,113 @@
+        period: '225 Earth days',
+        position: '02'
+    },
+    {
+        heading: 'Earth',
+        paragraph: 'Earth is the third planet from the Sun and the only astronomical object known to harbor life. About 29% of Earth\'s surface is land consisting of continents and islands. The remaining 71% is covered with water, mostly by oceans but also lakes, rivers and other fresh water, which together constitute the hydrosphere.',
+        period: '365 Earth days',
+        position: '03'
+    },
+    {
+        heading: 'Mars',
+        paragraph: 'The fourth planet from the Sun and the second-smallest planet in the Solar System, being larger than only Mercury. Mars carries the name of the Roman god of war and is often referred to as the "Red Planet". The days and seasons are comparable to those of Earth, because the rotational period as well as the tilt of the rotational axis relative to the ecliptic plane are similar.',
+        period: '687 Earth days',
+        position: '04'
+    }
+];
+
+
+document.querySelector("#menu-btn").addEventListener('click', function display(e) {
+    document.querySelector("#menu").classList.toggle("visible");
+});
+
+document.querySelector("#menu").addEventListener('click', function display(e) {
+    let sectionName = e.target.innerHTML.toLowerCase(),
+        target = e.target;
+    index = CONTENT.findIndex(el => el.heading.toLowerCase() == sectionName);
+    setData(index);
+});
+
+let addAnimation = (element, ...animationOptions) => element.animate(...animationOptions);
+
+let updateBackground = (element, path, id) => {
+    element.style.backgroundImage = window.innerWidth < 800 ? `url('${path + id + "_small"}.jpg')` : `url('${path + id}.jpg')`;
+    addAnimation(element, [{
+        opacity: .1
+    }, {
+        opacity: 1
+    }], {
+        duration: 500
+    });
+}
+
+let setInnerData = (element, data) => element.innerHTML = data;
+
+let setData = (index) => {
+    let image = document.getElementById("tabImage"),
+        heading = document.querySelector(".tab__header"),
+        headerMenu = document.querySelector(".header"),
+        container = document.querySelector(".container"),
+        paragraph = document.querySelector(".tab__paragraph"),
+        period = document.querySelector(".statistics__text"),
+        position = document.querySelector("#position");
+
+    updateBackground(image, ASSETS_PATH, (CONTENT[index].heading).toLowerCase());
+    setInnerData(heading, CONTENT[index].heading);
+    setInnerData(position, CONTENT[index].position);
+    setInnerData(paragraph, CONTENT[index].paragraph);
+    addAnimation(container, [{
+        opacity: 0
+    }, {
+        opacity: 1
+    }], {
+        duration: (600)
+    });
+
+    setInnerData(period, CONTENT[index].period);
+    if (window.innerWidth > 800) {
```

Comment:
It's better to move `800` to constants

Reply:
Created `const` in the top section of code for that value